example_scripts/SEAsia_DIC_example_plot.py

```diff
@@ -0,0 +1,38 @@
+"""
+seasia_r12_example_plot_BGC.py
+
+Make simple SEAsia 1/12 deg DIC plot.
+
+"""
+#%%
+import coast
+import matplotlib.pyplot as plt
+
+
+#################################################
+#%%  Loading  data
+#################################################
+path_examples = "/scratch/accord/COAST/coast_demo/COAsT_example_files/EXTRA_examples/"  ## data local
```

Comment:
Can you edit the path:
```
path_examples = "/scratch/accord/COAST/coast_demo/COAsT_example_files/EXTRA_examples/"
```
I've copied the files there. This is a more permanent store

Reply:
Just a thought but do we want absolute paths? I'm doing my testing on Jasmin (good for us to test on different platforms) so this would break it for me. Although maybe for scripts it is fine?

Reply:
Ok I have changed the path to in the example to: /projectsa/COAsT/NEMO_example_data/SEAsia_R12